nservicebus/endpoint/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Endpoint
+summary: Describe the concepts of Endpoint and Endpoint Instance.
+tags:
+- Endpoint
+- Endpoint Instance
+related:
+ - samples/endpoint-configuration
+---
+
+An Endpoint is a logical entity that communicates with other Endpoints via [Messaging](/nservicebus/messaging). Each Endpoint has an identifying name and contains a collection of Message Handlers and Sagas. An Endpoint can be deployed to a number of machines and environments. Each such deployment is a separate Endpoint Instance. Each Endpoint Instance has an [Input Queue](/nservicebus/endpoint/specify-input-queue-name.md) which contains messages for the Endpoint Instance to process. 
+
+It is common for each Endpoint to have a single Endpoint Instance. As Endpoints need to [scale-out](/nservicebus/scalability-and-ha/scale-out.md), additional Endpoint Instances can be added. This collection of Endpoint Instances still represents a single logical Endpoint. 
+
+NOTE: In a scale-out scenario multiple Endpoint Instances can share a single Input Queue. 
```

Comment:
As above, it isn't. And the sentence only tells that "can share". I honestly believe that this should be described in some other place. I left this sentence here.